e2e/testdata/fn-eval/entrypoint-failure/.expected/config.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+exitCode: 1
+testType: eval
+image: gcr.io/kpt-fn/set-namespace:v0.1.3
+entrypoint: function1
+args:
+  namespace: staging
+stdErr: |
+  [RUNNING] "gcr.io/kpt-fn/set-namespace:v0.1.3[function1]"
+  [FAIL] "gcr.io/kpt-fn/set-namespace:v0.1.3[function1]" in 0s
```

Comment:
It is not conveying what was the failure.

Reply:
So, failure is conveyed in the error message that get printed after that. Currently it relays the failure returned by the docker runtime because if it is all in string form. I had the message here earlier but that includes docker runtime specific info that varies across our dev/CI environments and that were making this test flaky, so removed the message.
I will check if we can find a better way to handle that.